### ISSUE TITLE
Figure out "best" image

### ISSUE THREAD

**Issue by siznax** [siznax/wptools#14]
We get the following images from MediaWiki:API and Wikidata:

```
Image => Wikidata Property:P18
pageimage => MediaWiki:API (action=query) pageimage
thumbnail => MediaWiki:API (action=query) thumbnail
```

RESTBase ``/page/mobile-text`` also gives us a couple of images:

```
image => apparently, pageimage (above)
thumb => apparently, scaled (larger) thumbnail (above)
```

Spot-checking some articles, there were only conflicts with ``thumbnail``, where the RESTBase ``thumb`` was the scaled (larger) MediaWiki:API ``thumbnail``.

In the case of _Napoleon_, the Wikidata ``Image`` is a portrait (great!), while the ``get_query`` images are an _Imperial Coat of Arms_ (what?).  

››› Napoleon (en)
* Image: https://upload.wikimedia.org/wikipedia/commons/b/b5/Jacques-Louis_David_-_The_Emperor_Napoleon_in_His_Study_at_the_Tuileries_-_Google_Art_Project_2.jpg
* images: <dict(5)> {Image, image, pageimage, thumb, thumbnail}
* pageimage: https://upload.wikimedia.org/wikipedia/commons/f/f3/Grandes_Armes_Imp%C3%A9riales_%281804-1815%292.svg
* thumbnail: https://upload.wikimedia.org/wikipedia/commons/thumb/f/f3/Grandes_Armes_Imp%C3%A9riales_%281804-1815%292.svg/882px-Grandes_Armes_Imp%C3%A9riales_%281804-1815%292.svg.png

In the case of _Stephen Fry_, ``pageimage`` seems more current.

››› Stephen Fry (en)
* Image: <https://upload.wikimedia.org/wikipedia/commons/1/15/Stephen_Fry_cropped.jpg>
* images: <dict(5)> {Image, image, pageimage, thumb, thumbnail}
* pageimage: <https://upload.wikimedia.org/wikipedia/commons/3/38/Stephen_Fry_June_2016.jpg>
* thumbnail: <https://upload.wikimedia.org/wikipedia/commons/3/38/Stephen_Fry_June_2016.jpg>

In the case of _Ella Fitzgerald_, it's hard to say.

››› Ella Fitzgerald (en) {}
* Image: https://upload.wikimedia.org/wikipedia/commons/6/65/Ella_Fitzgerald_1962.JPG
* images: <dict(5)> {Image, image, pageimage, thumb, thumbnail}
* pageimage: https://upload.wikimedia.org/wikipedia/commons/a/a1/Ella_Fitzgerald_(Gottlieb_02871).jpg
* thumbnail: https://upload.wikimedia.org/wikipedia/commons/thumb/a/a1/Ella_Fitzgerald_%28Gottlieb_02871%29.jpg/969px-Ella_Fitzgerald_%28Gottlieb_02871%29.jpg

So, which is the "best" image? I guess it depends on your goal.

**Comment by siznax** [comment]
A better way to organize these attributes may be...

```
qimage: MediaWiki:API pageimage
qthumb: MediaWiki:API thumbnail
rimage: RESTBase image
rthumb: RESTBase thumb
wimage: Wikidata Property:P18
```

**Comment by siznax** [comment]
Mostly fixed with d5577c3: make images: {qimage, qthumb, rimage, rthumb, wimage}
